bwt/bwt_test.go

```diff
@@ -0,0 +1,215 @@
+package bwt
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"testing"
+
+	"golang.org/x/exp/slices"
+)
+
+func ExampleBWT_Count() {
+	inputSequence := "AACCTGCCGTCGGGGCTGCCCGTCGCGGGACGTCGAAACGTGGGGCGAAACGTG"
+
+	bwt, err := New(inputSequence)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	fmt.Println(bwt.Count("CG"))
+	// Output: 10
+}
+
+type BWTCountTestCase struct {
+	seq      string
+	expected int
+}
+
+func TestBWT_Count(t *testing.T) {
```

Comment:
Is this case just making a substring histogram?

Reply:
I'm actually not sure what Count could be used for in the real world tbh.

I'm thinking that it could be a quick way to check for the existence of an exact match? That or for statistics like you mentioned?

Might be a good question for the community- is this useful functionality to expose?

Reply:
@Koeng101 is the Count of sub-sequences useful to Expose?

Reply:
Yes, it is! There are instances where you are looking for multiple counts of sub-sequences (like restriction enzyme sites). Generally speaking, useful to have counts.